WordPress/Tests/NamingConventions/PrefixAllGlobalsUnitTest.1.inc

```diff
@@ -439,4 +441,10 @@ function aa_do_something(){}
 
 // The following line mimicks an empty prefix value.
 // phpcs:set WordPress.NamingConventions.PrefixAllGlobals prefixes[] ,
-function aa_do_something(){}
\ No newline at end of file
+function aa_do_something(){}
+
+// phpcs:set WordPress.NamingConventions.PrefixAllGlobals prefixes[] 😊
+function 😊_do_something(){}
+
+// phpcs:set WordPress.NamingConventions.PrefixAllGlobals prefixes[] 😊😊
+function 😊😊_do_something(){}
```

Comment:
Clear line space needed at end of file.

Reply:
This should've been added in this commit: https://github.com/WordPress/WordPress-Coding-Standards/pull/1741/commits/bbe5ef96d6601a0de159ab20f42dfb48c63c0a53